src/ucp/api/ucp.h

```diff
@@ -169,8 +186,37 @@ enum ucp_ep_params_field {
                                                             peer */
     UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE = UCS_BIT(1), /**< Error handling mode.
                                                             @ref ucp_err_handling_mode_t */
-    UCP_EP_PARAM_FIELD_ERR_HANDLER       = UCS_BIT(2)  /**< Handler to process
+    UCP_EP_PARAM_FIELD_ERR_HANDLER       = UCS_BIT(2), /**< Handler to process
                                                             transport level errors */
+    UCP_EP_PARAM_FIELD_SOCK_ADDR         = UCS_BIT(3), /**< Using a client-server
+                                                            connection establishment
+                                                            mechanism for establishing
```

Comment:
Why is it incorrect?
In the previous CR you asked to include which field it requires.
`need to write which field is required (sockaddr)`
"Socket address field" is what you meant by this?